.github/workflows/test.yml

```
@@ -22,7 +22,7 @@ jobs:
       - name: Run Checks
         run: |
           gem install awesome_bot
-          awesome_bot -f README.md --allow-ssl --allow 302,429
+          awesome_bot -f README.md --allow-ssl --allow 302,429 --whitelist firsttimersonly,otherdomain-without-the-TLD
```

If this is commit is supposed to address [this comment](https://github.com/freeCodeCamp/how-to-contribute-to-open-source/issues/432#issuecomment-702941295) on duplicates, we can simply add the flag that'll allow duplicates.

```suggestion
          awesome_bot -f README.md --allow-ssl --allow 302,429 --allow-dupe
```

[For reference](https://github.com/dkhamsing/awesome_bot#command-line). This flag will be easier and more appropriate for duplicates rather than having to manually add every problematic website.

---

My understanding is the workflow *should* catch duplicates (to avoid duplicate addition of resources), but there are some erroneous matches to filter out.

---

Oh, I see. That makes sense.

If that's the case, then I'd make this suggestion.

```suggestion
          awesome_bot -f README.md --allow-ssl --allow 302,429 --white-list firsttimersonly.com,github.com/github,creativecommons.org/licenses/by-sa/4.0
```